.changeset/large-spies-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': patch
```

Comment:
Technically a minor I guess, since it's a 1.x package

Reply:
Ah OK, I'll update this. I wasn't sure since the CONTRIBUTING doc said minor was for breaking changes.

Reply:
If it's a 0.x package yeah. For Things that have passed the 1.0 marker, regular semver rules apply.